### ISSUE TITLE
[Feature Request] VRChat Chatbox custom display time

### ISSUE THREAD

**Issue by TayouVR** [Curtis-VL/OVRToolkit-Issues#153]
Some OSC apps send a empty message to the chatbox to clear it. that way The user listening to music e.g. could have the chatbox only appear max. 5 or 10 seconds instead of the default 30 defined in the client.
OSC apps like https://github.com/PaciStardust/HOSCY also send that clear message after regular text based on how log the text is (it'll show e.g. "Yes" for 3 seconds and "Hello, how are you doing?" for 10)

Though a basic max. display time for the media thing would be neat already

**Comment by Curtis-VL** [comment]
This works? It clears the box...?

If so, that's great and soooooooo much better than a message appearing for 30 seconds.

**Comment by nullstalgia** [comment]
Yes. The VRC OSC Chatbox app that came before yours had a button to clear, if you want a practical example.

https://github.com/Gosha/OVRTK-VRC-Chatbox-Input/blob/master/VRC-Input/html/input.js#L16
